File: Easy/412. fizzbuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        answer = []
        for i in range(1, n+1):
            temp = ''
            if i % 3 == 0:
                temp += 'Fizz'
            if i % 5 == 0:
                temp += 'Buzz'
            if i % 3 != 0 and i % 5 != 0:
                temp += f'{i}'
            answer.append(temp)
        return answer

File: Easy/412.fizzbuzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> answer;
        for (int i = 1; i <= n; i++) {
            string temp = "";

            if (i % 3 == 0 && i % 5 == 0) {
                temp = "FizzBuzz";
            } else if (i % 3 == 0) {
                temp = "Fizz";
            } else if (i % 5 == 0) {
                temp = "Buzz";
            } else {
                temp = to_string(i);
            }
            answer.push_back(temp);
        }
        return answer;
    }
};